store/mongo.go

```diff
@@ -13,11 +13,13 @@ const (
 )
 
 type DataStorePool interface {
+	RegisterAccount(accountNumber string) error
```

Comment:
It seems that this functions is not used.

`indexForPersonalAccountStore` is called in `Account`, which returns a personal data store.

Reply:
It is not in-used. I left it intentionally. It is fine to remove.